src/views/errors/PageNotFoundView.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
I think we shall structure ower folder like this:
- src/pages
- src/components

all components are imported in a certain page and the page is going to be used in routes